playerbot/PlayerbotMgr.cpp

```diff
@@ -47,25 +63,31 @@ void PlayerbotHolder::UpdateSessions(uint32 elapsed)
             LogoutPlayerBot(bot->GetObjectGuid().GetRawValue());
         }
     }
+
+   Cleanup();
+}
+
+void PlayerbotHolder::Cleanup()
+{
+   auto it = playerBots.begin();
+
+   while (it != playerBots.end())
+   {
+      if (it->second == nullptr)
+      {
+         it = playerBots.erase(it);
+      }
+
+      ++it;

```

Comment:
I believe ++it should be when not deleting

Reply:
yeah, there's continue inside, wrong cherry pick